doc/lua_api.md

```diff
@@ -2191,6 +2191,79 @@ Table of resulting tool uses:
 
 
 
+Wear Bar Color
+==============
+
+'Wear Bar' is a property of items that defines the coloring
+of the bar that appears under damaged tools.
+If it is absent, the old behavior of green-yellow-red is
+used.
+
+Wear bar colors definition
+--------------------------
+
+### Syntax
+
+For non-blending mode:
+```lua
+{
+    {
+        color = "#ff00ff",
+        min_durability = 0.2, -- inclusive
+        max_durability = 0.3  -- exclusive
+    },
+    {
+        color = "#c0ffee",
+        min_durability = 0.45,
+        max_durability = 0.6
+    },
+    -- color to use if no other ranges match the durability
+    default = "#ffff00",
+    blend = false
+}
+```
+
+For blending mode:
+```lua
+{
+	-- specify color for a specific durability percent, and blend between them
+    [0.2] = "#ff00ff",
+    [0.45] = "#c0ffee",
+	-- used for 0% and 100% durability if no color explicitly specified
+    default = "#ffff00",
+    blend = true
```

Comment:
Sorry, I made a mistake when writing the suggestion. It should be all spaces (4 spaces for one indentation).
```suggestion
    -- specify color for a specific durability percent, and blend between them
    [0.2] = "#ff00ff",
    [0.45] = "#c0ffee",
    -- used for 0% and 100% durability if no color explicitly specified
    default = "#ffff00",
    blend = true
```